setup/steps/check_replication_status.go

```diff
@@ -0,0 +1,23 @@
+package steps
+
+import (
+	"errors"
+
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var CheckReplicationStatus = &s.Step{
+	Description: "Check replication status",
+	Check: func(state *s.SetupState) (bool, error) {
+		result, err := state.QueryRunner.QueryRow("SELECT pg_is_in_recovery()")
+		if err != nil {
+			return false, err
+		}
+		isReplicationTarget := result.GetBool(0)
+
+		if isReplicationTarget {
+			return false, errors.New("not supported for replicas")
```

Comment:
maybe slightly more clear if we word it as `Postgres server is a replica - this is currently not supported in this setup`